sessions/forecasting-concepts.qmd

```diff
@@ -171,77 +171,81 @@ first_forecast |>
- - The uncertainty grows for a few horizons and then stays constant. 
- 
-This is a pretty bad forecast, but here are a few features of the forecast that pass the sniff test:
+-   The fact that some of the prediction intervals have negative values. The percent of outpatient visits due to ILI cannot be negative! That doesn't pass the sniff test.
+-   The mean predictions go up and then level off. There is no notion of a "seasonal peak" in the forecast.
+-   The uncertainty grows for a few horizons and then stays constant.
 
- - For the first few horizons, the uncertainty gets wider.
- - For the first few horizons, the predictions go up, which tracks with the recent data trends, and what we might expect from flu season in the early fall.
- - The mean predictions and prediction intervals are smooth. Unless there is good reason for forecasts to be jagged, it tends to be the sign of a model that has not been overfit to have smooth-ish predictions. (Of course, it might still be a bad forecast.)
-::::
+This is a pretty bad forecast, but here are a few features of the forecast that pass the sniff test:
 
+-   For the first few horizons, the uncertainty gets wider.
+-   For the first few horizons, the predictions go up, which tracks with the recent data trends, and what we might expect from flu season in the early fall.
+-   The mean predictions and prediction intervals are smooth. Unless there is good reason for forecasts to be jagged, it tends to be the sign of a model that has not been overfit to have smooth-ish predictions. (Of course, it might still be a bad forecast.)
+:::
 
-Can you brainstorm some ideas to help this model improve? This will be the topic of the upcoming sessions.
+Can you brainstorm some ideas to help this model improve?
+This will be the topic of the upcoming sessions.
 
 ## Anatomy of a fable forecast
 
-We will dive into this in more detail later in the course, but it might be interesting now to look under the hood of what format `fable` uses to store the forecast itself. Check it out:
+We will dive into this in more detail later in the course, but it might be interesting now to look under the hood of what format `fable` uses to store the forecast itself.
+Check it out:
 
 ```{r}
 class(first_forecast)
 first_forecast
 ```
 
-The `first_forecast` object is `fbl_ts` which is a special kind of `tsibble`, which is in turn a special kind of `data.frame`. 
-In this case, each horizon for which we've asked for a prediction has a row in the dataset. 
-There is a `.mean` column where you can quickly see the predicted mean, and a `wili` column which contains values of class `dist`. 
+The `first_forecast` object is `fbl_ts` which is a special kind of `tsibble`, which is in turn a special kind of `data.frame`.
+In this case, each horizon for which we've asked for a prediction has a row in the dataset.
+There is a `.mean` column where you can quickly see the predicted mean, and a `wili` column which contains values of class `dist`.
```

Comment:
did you use a lintr to remove the trailing spaces here? I feel like I'm looking at a lot of lines that haven't really been changed.